spds/triggers.markdown

```diff
@@ -9,11 +9,11 @@ permalink: /triggers/
-| [RAMUtilizationTrigger](../docu/#deunistuttgartsqaslingshotspdtriggersRAMUtilizationTrigger)                                | A utilization based trigger based on the RAM resource.                               |  [ProcessingResourceUtilizationBasedTrigger](../docu/#deunistuttgartsqaslingshotspdtriggersProcessingResourceUtilizationBasedTrigger)  |
-| [IdleTimeTrigger](../docu/#deunistuttgartsqaslingshotspdtriggersIdleTimeTrigger) | IdleTimeTrigger bases the trigger on the idle time of a resource container.          | [TimeBasedTrigger](../docu/#deunistuttgartsqaslingshotspdtriggersTimeBasedTrigger)                     |
-| [ResponseTimeTrigger](../docu/#deunistuttgartsqaslingshotspdtriggersResponseTimeTrigger)                              | A trigger based on the response time exceeding a reference threshold value.          |  [TimeBasedTrigger](../docu/#deunistuttgartsqaslingshotspdtriggersTimeBasedTrigger)                                   |
-| [PointInTimeTrigger](../docu/#deunistuttgartsqaslingshotspdtriggersPointInTimeTrigger)                             | A trigger that is defined by the time on which a spd.TargetGroup should be adjusted. | [Scaling Trigger](../docu/#deunistuttgartsqaslingshotspdtriggersScalingTrigger)                         |
+| [CPUUtilizationTrigger](../docu/#triggersCPUUtilizationTrigger)                                  | A utilization based trigger based on the CPU resource.                               | [ProcessingResourceUtilizationBasedTrigger](../docu/#triggersProcessingResourceUtilizationBasedTrigger) |
+| [HDDUtilizationTrigger](../docu/#triggersHDDUtilizationTrigger)                                  | A utilization based trigger based on the HDD resource.                               |  [ProcessingResourceUtilizationBasedTrigger](../docu/#triggersProcessingResourceUtilizationBasedTrigger)  |
+| [RAMUtilizationTrigger](../docu/#triggersRAMUtilizationTrigger)                                | A utilization based trigger based on the RAM resource.                               |  [ProcessingResourceUtilizationBasedTrigger](../docu/#triggersProcessingResourceUtilizationBasedTrigger)  |
+| [IdleTimeTrigger](../docu/#triggersIdleTimeTrigger) | IdleTimeTrigger bases the trigger on the idle time of a resource container.          | [TimeBasedTrigger](../docu/#triggersTimeBasedTrigger)                     |
+| [ResponseTimeTrigger](../docu/#triggersResponseTimeTrigger)                              | A trigger based on the response time exceeding a reference threshold value.          |  [TimeBasedTrigger](../docu/#triggersTimeBasedTrigger)                                   |
+| [PointInTimeTrigger](../docu/#triggersPointInTimeTrigger)                             | A trigger that is defined by the time on which a spd.TargetGroup should be adjusted. | [Scaling Trigger](../docu/#triggersScalingTrigger)                         |
```

Comment:
These links are still broken, probably related to my previouse comment.

Reply:
These are broken because the whole triggering part was restructured ... I would commit another PR for revising the triggering